src/documents/migrations/0023_document_searchable_content.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 2.0.13 on 2019-05-30 14:50
+import unicodedata
+
+from django.db import migrations, models
+
+from paperless.utils import slugify as slugifyOCR
+
+
+class Migration(migrations.Migration):
+    dependencies = [("documents", "0022_auto_20181007_1420")]
+
+    reversible = True
+
+    def casefold_forwards(apps, schema_editor):
+        Document = apps.get_model("documents", "Document")
+        for doc in Document.objects.all():
+            if doc.title is not None:
+                doc.searchable_title = slugifyOCR(doc.title)
+            if doc.content is not None:
+                doc.searchable_content = slugifyOCR(doc.content)
+            doc.save()
+
+    def casefold_backwards(apps, schema_editor):
```

Comment:
Protip: you don't need to create an empty backwards method.  You can just do `migrations.RunPython(casefold_forwards, migrations.RunPython.noop)` below instead.